clmm/utils/boost.py

```diff
@@ -1,24 +1,39 @@
 """General utility functions that are used in multiple modules"""
+
 import numpy as np
 
 
-def compute_nfw_boost(rvals, rscale=1000, boost0=0.1):
-    """Given a list of `rvals`, and optional `rscale` and `boost0`, return the corresponding
-    boost factor at each rval
+def compute_nfw_boost(rvals, rscale, boost0=0.1):
+    r"""Computes the boost factor at radii rvals using a parametric form 
+    following that of the analytical NFW excess surface density.
+
+    Setting :math:`x = R/R_s` 
+
+    .. math::
+        B(x) = 1 + B_0\frac{1-F(x)}{(x)^2 -1}
+
+    where
+
+    .. math::
+        F(x) = \begin{cases} \frac{\arctan \sqrt{x^2 -1 }}{\sqrt{x^2 -1 }} & \text{if $x>1$}, \\
+        \text{1} & \text{if x = 1}, \\
+        \frac{\text{arctanh} \sqrt{1-x^2 }}{\sqrt{1- x^2 }} & \text{if $x<1$}.
```

Comment:
In the actual calculation (`_calc_finternal` below), the arctan and arctanh is inside a natural log.

Reply:
I meant to ask who coded up the boost, but my guess is that the complex notation with natural log used in the code ends up being equivalent to explicitely writing things in terms of arctan and arctanh. I half-convinced myself this was the case but didn't spend enough time on this. Would be good to have confirmation

Reply:
Wolfram alpha says they are the same expression. Pretty cool.
https://www.wolframalpha.com/input?i=arctanh%28sqrt%28x%5E2-1%29%29%2Fsqrt%28x%5E2-1%29